### PR TITLE
CASMPET-6287 add comment about not restarting DVS clients and servers at the same time

### PR DESCRIPTION
**Pull request by leliasen-hpe**
# Description

CASMPET-6287 add comment about not restarting DVS clients and servers at the same time

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ make sure that the following conditions are met:
 
 - No single rebuild request should include all of the worker nodes that have DVS running on them.  For High Availability, DVS requires at least two workers running DVS and CPS at all times.
 
+- It is not recommended to reboot compute nodes at the same time as workers nodes with DVS servers are being rebuilt. This is to avoid restarting DVS clients and servers at the same time.
+
```
Comment (djlapenta) [inline operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md:57]: When rebuilding worker nodes which are running DVS, it is not recommended to simultaneously reboot compute nodes.

File: upgrade/Stage_2.md [hunk 2]
```diff
@@ -70,6 +70,8 @@ cray cfs components update --state '[]' <XNAME>
 
 ## Stage 2.2 - Worker node image upgrade
 
+> **`NOTE`** It is not recommended to reboot compute nodes at the same time as workers nodes with DVS servers are being upgraded. This is to avoid restarting DVS clients and servers at the same time.
+
```
Comment (djlapenta) [inline upgrade/Stage_2.md:74]: same thing - minor change suggestion..

When rebuilding worker nodes which are running DVS, it is not recommended to simultaneously reboot compute nodes.

**Review by rgross727 — APPROVED** [review]

**Review by bo-quan — APPROVED** [review]

**Pull request merged**